src/lib/utils/fonts.ts

```diff
@@ -60,16 +59,52 @@ const FONT_CONFIG: Record<
   },
 };
 
+const CYRILLIC_FONT_CONFIG: Record<Font, FontSettings> = {
```

Comment:
Added additional font config since Cyrillic fonts have different sizes and do not align with sizes provided for Latin fonts